iabtcf-decoder/src/main/java/com/iabtcf/utils/FieldDefs.java

```diff
@@ -194,7 +194,6 @@ protected boolean isDynamic() {
             isDynamic = offset.isDynamic() || length.isDynamic();
             isDynamicInit = true;
         }
-
```

Comment:
Should this method be synchronized as the comments (on the abstract method) say it should be thread safe. Do you think it's ok for two threads to enter the if loop, which I think is possible in this implementation.

Reply:
This method is thread safe, it doesn't need to be synchronized.

Reply:
to be more clear, it doesn't matter if multiple threads concurrently execute this method. the result will be the same. no need for additional synchronization constructs.

Reply:
Sounds good.